editor/plugins/org.fusesource.ide.branding/META-INF/MANIFEST.MF

```diff
@@ -7,7 +7,8 @@ Bundle-Vendor: %Bundle-Vendor
 Bundle-RequiredExecutionEnvironment: JavaSE-1.7
 Bundle-Activator: org.fusesource.ide.branding.Activator
 Bundle-ActivationPolicy: lazy
-Export-Package: org.fusesource.ide.branding
+Export-Package: org.fusesource.ide.branding,
+ org.fusesource.ide.branding.perspective
```

Comment:
i think that it is used only for test purpose, so better to put a x-friends to restrict visibility